### PR TITLE
Firefox iOS 9.0 String Export (2017-08-17)

### PR DESCRIPTION
**Pull request by st3fan**

### CODE REVIEW

File: en-US/firefox-ios.xliff [hunk 1]
```diff
@@ -1132,6 +1121,96 @@
...
+      </trans-unit>
+      <trans-unit id="FxA.AccountVerificationRequired">
+        <source>Account Verification Required</source>
+        <target>Account Verification Required</target>
+        <note>Label stating your account is not verified</note>
+      </trans-unit>
+      <trans-unit id="FxA.AccountVerificationRequiredSurface">
+        <source>You need to verify %@. Check your email for the verification link from Firefox.</source>
+        <target>You need to verify %@. Check your email for the verification link from Firefox.</target>
+        <note>Message explaining that user needs to check email for Firefox Account verfication link.</note>
```
Comment (flodolo) [inline en-US/firefox-ios.xliff]: Not important, but typo at the end of the comment

File: en-US/firefox-ios.xliff [hunk 2]
```diff
@@ -1132,6 +1121,96 @@
...
+        <target>Wrong email? Disconnect below to start over.</target>
+        <note>Label stating how to disconnect account</note>
+      </trans-unit>
+      <trans-unit id="FxA.AccountVerificationRequired">
+        <source>Account Verification Required</source>
+        <target>Account Verification Required</target>
+        <note>Label stating your account is not verified</note>
+      </trans-unit>
+      <trans-unit id="FxA.AccountVerificationRequiredSurface">
+        <source>You need to verify %@. Check your email for the verification link from Firefox.</source>
```
Comment (flodolo) [inline en-US/firefox-ios.xliff]: The verification link is not really from Firefox (it's from Firefox Account), but I guess there was copy review for this.

File: en-US/firefox-ios.xliff [hunk 3]
```diff
@@ -1132,6 +1121,96 @@
...
+        <target>Sync Now</target>
+        <note>Button label to Sync your Firefox Account</note>
+      </trans-unit>
+      <trans-unit id="FxA.TakeYourWebWithYou">
+        <source>Take Your Web With You</source>
+        <target>Take Your Web With You</target>
+        <note>Call to action for sign into sync button</note>
+      </trans-unit>
+      <trans-unit id="FxAPush_DeviceConnected_body">
+        <source>Firefox Sync has connected to %@</source>
```
Comment (flodolo) [inline en-US/firefox-ios.xliff]: I'm confused by this string: "Firefox Sync has connected **to** X" but "Firefox Sync has disconnected X" later? 

In general, it sounds really weird compared to something like "X is now connected to Firefox Sync". Proof is that the comment explains it differently (and more clearly).

Also, no connection message for device with unknown name?

File: en-US/firefox-ios.xliff [hunk 4]
```diff
@@ -1874,6 +1953,36 @@
...
+        <target>Always</target>
+        <note>DNT Settings option for always on</note>
+      </trans-unit>
+      <trans-unit id="Settings.DNT.OptionOnWithTP">
+        <source>Only when using Tracking Protection</source>
+        <target>Only when using Tracking Protection</target>
+        <note>DNT Settings option for only turning on when Tracking Protection is also on</note>
+      </trans-unit>
+      <trans-unit id="Settings.DNT.Title">
+        <source>Send websites a “Do Not Track” signal that you don’t want to be tracked</source>
```
Comment (flodolo) [inline en-US/firefox-ios.xliff]: You're mixing up proper apostrophes and straight quotes in strings. This string uses ’, for example intro.Slides.Private.Title uses '.

File: en-US/firefox-ios.xliff [hunk 5]
```diff
@@ -1969,6 +2078,16 @@
         <target>New Tab Settings</target>
         <note>Title displayed in header of the setting panel.</note>
       </trans-unit>
+      <trans-unit id="Settings.OfferClipboardBar.Status">
+        <source>When opening Firefox</source>
+        <target>When opening Firefox</target>
+        <note>Status text of setting to enable the Go to Copied URL feature - https://bug1223660.bmoattachments.org/attachment.cgi?id=8898349</note>
```
Comment (flodolo) [inline en-US/firefox-ios.xliff]: While the screenshot is great, an additional explanation that avoids leaving Pontoon while translating is even better. E.g.: `Description displayed under the ”Offer to open copied link” option.`

File: en-US/firefox-ios.xliff [hunk 6]
```diff
@@ -1969,6 +2078,16 @@
...
+      </trans-unit>
+      <trans-unit id="Settings.OfferClipboardBar.Title">
+        <source>Offer to open copied links</source>
```
Comment (flodolo) [inline en-US/firefox-ios.xliff]: Is the case correct? It doesn't look like, at least looking at the image.

**Comment by st3fan** [comment]
@Delphine let's wait with merging - i will address the things that @flodolo reported.

File: en-US/firefox-ios.xliff (hunk 2 above)
Comment (st3fan) [inline en-US/firefox-ios.xliff]: Asking @vbudhram for advice

File: en-US/firefox-ios.xliff (hunk 3 above)
Comment (st3fan) [inline en-US/firefox-ios.xliff]: Asking @vbudhram for advice

File: en-US/firefox-ios.xliff (hunk 2 above)
Comment (vbudhram) [inline en-US/firefox-ios.xliff]: > but I guess there was copy review for this.

Unfortunately, this particular string didn't get a copy review. I added it to make sure I had something along the lines for the string freeze. FWIW, this string probably not get used in 9.0.

File: en-US/firefox-ios.xliff (hunk 1 above)
Comment (vbudhram) [inline en-US/firefox-ios.xliff]: I think this is ok, still it won't be surfaced to user.

File: en-US/firefox-ios.xliff (hunk 3 above)
Comment (vbudhram) [inline en-US/firefox-ios.xliff]: Should it be `Firefox Sync has connected from X`?

Comment (st3fan) [inline en-US/firefox-ios.xliff]: @vbudhram let's ask rfeeley, maybe there is a good string we can take from Desktop?

Comment (ryanfeeley) [inline en-US/firefox-ios.xliff]: @st3fan @flodolo @vbudhram Please always use the strings from desktop. https://hg.mozilla.org/mozilla-central/file/default/browser/locales/en-US/chrome/browser/accounts.properties

Comment (ryanfeeley) [inline en-US/firefox-ios.xliff]: Ah, so iOS is adding a notification that is not in desktop. I'd prefer that we don't show this notification, but if we have to the copy can be:

**Firefox Sync**
%S has been successfully disconnected.

**Comment by st3fan** [comment]
I'll create a new PR with more changes.

**Pull request closed without merging**